src/Infrastructure/Event/ApiExceptionListener.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Event;
+
+use Codeception\Util\HttpCode;
+use SprykerSdk\Sdk\Infrastructure\Exception\InvalidRequestDataException;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpKernel\Event\ExceptionEvent;
+
+class ApiExceptionListener
+{
+    /**
+     * @param \Symfony\Component\HttpKernel\Event\ExceptionEvent $event
+     *
+     * @return void
+     */
+    public function onKernelException(ExceptionEvent $event): void
+    {
+        $exception = $event->getThrowable();
+
+        if ($exception instanceof InvalidRequestDataException) {
+            $response = new JsonResponse(
+                [
+                    'message' => $exception->getMessage(),
+                ],
+                HttpCode::NOT_FOUND,
```

Comment:
Use `Symfony\Component\HttpFoundation\Response::HTTP_NOT_FOUND` instead of `Codeception\Util\HttpCode`